app/src/main/java/com/github/pedrovgs/effectiveandroidui/ui/activity/MainActivity.java

```diff
@@ -54,7 +53,7 @@ private void initializeTvShowFragment() {
     }
 
     private void initializeTvShowDraggableFragment() {
-        tvShowDraggableFragment =
+        TvShowDraggableFragment tvShowDraggableFragment =
```

Comment:
PMD is not very happy if you have a private field which could be used just as a local variable.